### ISSUE TITLE
Observation information is not accurate

### ISSUE THREAD

**Issue by bels-ap** [HCL-TECH-SOFTWARE/domino-license-analysis-utility-DLAU#50]
I'm running the latest version 1.2 and am still seeing the following. 

In the Observation view, 4 entries are being reported. However the listing is not correct / does not reflect the actual detail information. When I click through the observations, I have two entries with a "Warnings" flag and two without any flag. 
![observations](https://github.com/HCL-TECH-SOFTWARE/domino-license-analysis-utility-DLAU/assets/56879557/b8b4f120-8894-4477-a45b-6da8cd309289)

**Comment by bels-ap** [comment]
this still applies in 1.2.1. 
The overview shows 4 Observations: 1 Security, 1 Warning, 2 Info Only
When opening the observations form, I see 2 Warnings and 2 Informational entries
Maybe the Security observation is regarding the use of LDAP but there is still a mismatch between the overview and the details in the form.

**Comment by notesboy** [comment]
@bels-ap - this fix is a "Go Forward" solution. Existing results may still have a skewed/incorrect message when selecting the Observations.

Please re-run the DLAU Scans using V1.2.1. The new results will have the correct message.

**Comment by bels-ap** [comment]
No, they do not. I did run a new scan (of course) and also created a new database with the 1.2.1 template, still the same result/mismatch.
